### ISSUE TITLE
What is the status of version 0.3?

### ISSUE THREAD

**Issue by ajtribick** [google/wuffs#134]
I see there's a 0.4 alpha version in releases/c. Is the 0.3 version still considered unstable (per the README.md file that states that 0.2 is the latest stable)?

**Comment by nigeltao** [comment]
If you're basing that on these lines in the README.md file:

> Status
>
> Version 0.3. The API and ABI aren't stabilized yet. 

"Aren't stabilized" refers to the 0 in "Version 0.X". Version 0.N (for N > 3) isn't guaranteed to be API or ABI compatible with 0.3. In contrast, there isn't a version 1.0 yet, but I'd expect 1.0 to be compatible with version 1.1 (but not with version 2.0).

Version 0.3 is just as stable as version 0.2. They are both very unlikely to change from here, other than security updates.

New features, new standard library packages, new API, API/ABI breakage, etc. all happen on the work-in-progress version 0.4 stream.

In summary: use version 0.3.

Sorry for the confusion.

**Comment by ajtribick** [comment]
Thanks for the clarification! I should have been more specific about which README.md I was referring to: it was the one in release/c:

> **Version 0.2 (December 2019)** is the latest stable version.

**Comment by nigeltao** [comment]
Ah, I forgot about that one. Thanks for the bug report.
